.github/workflows/test_suites.yml

```
@@ -162,6 +168,8 @@ jobs:
       relational-db-migration-tests,
       docker-compose-test,
       docker-ci-test,
+      integration-tests
```

actionlint (1.7.11)

[error] invalid job ID "integration-tests slow-e2e-tests". job ID must start with a letter or _ and contain only alphanumeric characters, -, or _

(id)